.coderabbit.yaml

```
@@ -8,3 +8,5 @@ ai_code_review:
     - node_modules/**
     - build/**
     - dist/**
+reviews:
+  review_status: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)